### PR TITLE
travis.yml: Fix build issues

### LINKED ISSUE #6133: test pipeline of travis.ci broken
Pipeline fails while installing dependencies.

### PR DESCRIPTION
**Pull request by anshalshukla**
This along with PR [passes autopep8](https://github.com/coala/coala/pull/6145) and [restrict bandit version](https://github.com/coala/coala-bears/pull/2984) should fix the travis.ci pipeline.

Will make changes in coala-moban once these are approved.

closes #6133

### CODE REVIEW

File: bear-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+radon==1.4.0
+restructuredtext-lint~=1.0
+rstcheck~=3.1
+safety~=1.8.2
+scspell3k~=2.0
+sqlparse~=0.2.4
+vim-vint~=0.3.12,!=0.3.19
+vulture~=0.25.0
+yamllint~=1.12.0
+yapf>=0.21.0<0.27.0
```
Comment (abhishalya) [inline bear-requirements.txt:42]: This file does not belong here. Where is this even used in this repository?

**Review by abhishalya — CHANGES_REQUESTED** [review]

**Comment by abhishalya** [comment]
@anshalshukla Can you please cleanup this PR, I've merged your coala-bears fix so you should be able to get a green build here ideally.

File: bear-requirements.txt (hunk 1 above)
Comment (anshalshukla) [reply bear-requirements.txt:42]: This doesn't seem right. I had cherry-picked the commits in the hope of making the builds green again. Yeah, it doesn't belong to this repo, and I'll make the necessary changes.

**Comment by anshalshukla** [comment]
Ideally, it should have, but it doesn't. One possible reason is that there is still some problem with bandit versioning. It goes on to install version 1.6.3, which is actually a yanked version.  I think fixing it to 1.6.2 should fix the pipeline. Should I make another PR for that on bears repo?

**Comment by abhishalya** [comment]
Yeah, you can do that. Also to check you can edit the coala bears zip url to your coala-bears fork instead so that we can see the resulting build here. Later we can remove that part from this PR.

**Comment by anshalshukla** [comment]
Okay, let me do that

**Comment by anshalshukla** [comment]
Yeah so that change makes the build (green)[https://github.com/anshalshukla/coala-bears].

**Comment by abhishalya** [comment]
@anshalshukla Awesome, will get this merged tomorrow :)

**Comment by anshalshukla** [comment]
Finally!! Merge [PR](https://github.com/coala/coala-bears/pull/2986) before this one.

File: .travis.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ jobs:
   include:
     - stage: sentinel
       # All other jobs will be cancelled if the sentinel job fails
-      python: 3.6
+      python: 3.6.9
```
Comment (abhishalya) [inline .travis.yml:43]: Why is this change required? It should work fine with any version of 3.6

File: .travis.yml [hunk 3]
```diff
@@ -77,7 +77,7 @@ env:
     - PATH="$PATH:$TRAVIS_BUILD_DIR/node_modules/.bin"
     - BEARS_ZIP_URL=https://codeload.github.com/coala/coala-bears/zip
     - GRAVIS="https://raw.githubusercontent.com/DanySK/Gravis-CI/master/"
-    - JDK="adopt@1.8.0-212"
+    - JDK="graalvm-ce-java8@21.0.0"
```
Comment (abhishalya) [inline .travis.yml:80]: Can you link from where you took this from?

File: .travis.yml [hunk 4]
```diff
@@ -107,7 +107,7 @@ before_install:
   - pip uninstall setuptools --yes || true
   - pip uninstall setuptools --yes || true
   - pip uninstall pipenv --yes || true
-  - pip install pip==9.0.1 setuptools==21.2.2
+  - pip install pip==9.0.3 setuptools==21.2.2
```
Comment (abhishalya) [inline .travis.yml:110]: Remember reason for this change?

**Review by abhishalya — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (anshalshukla) [reply .travis.yml:43]: tests module was failing on travis.ci with python version 3.6, you can have a look at [this](https://travis-ci.com/github/anshalshukla/coala/jobs/490860929). But it was working fine on my local machine with python3.6.9 and pip9.0.3 so I tried running the build using that configuration which worked (you can look at [this](https://travis-ci.com/github/anshalshukla/coala/jobs/490861059).

File: .travis.yml (hunk 3 above)
Comment (anshalshukla) [reply .travis.yml]: language-check package requires JAVA8([link](https://stackoverflow.com/questions/45725399/pip-install-language-check-not-working)). Among the list of JDK [supported ](https://travis-ci.com/github/anshalshukla/coala/jobs/492086200) I choose graalvm-ce-java8 as it is JAVA8 based and [open-source](https://www.graalvm.org/community/opensource/).

File: .travis.yml (hunk 2 above)
Comment (abhishalya) [reply .travis.yml:43]: Yeah, strange behavior. Might need to look a bit more in this. For now, this is fine.

File: .travis.yml (hunk 3 above)
Comment (abhishalya) [reply .travis.yml]: We can keep using adoptopenjdk. Don't want to switch to new one without any reason.

**Review by abhishalya — APPROVED** [review]
LGTM :tada: 

Thanks for fixing the build. Can you also do the moban changes now? That would give us a complete green build.
